DependencyInjection/AbstractSonataAdminExtension.php

```diff
@@ -43,7 +43,7 @@ protected function fixTemplatesConfiguration(array $configs, ContainerBuilder $c
                         'datetime' => 'SonataAdminBundle:CRUD:list_datetime.html.twig',
                         'text' => 'SonataAdminBundle:CRUD:list_string.html.twig',
                         'textarea' => 'SonataAdminBundle:CRUD:list_string.html.twig',
-                        'email' => 'SonataAdminBundle:CRUD:list_string.html.twig',
+                        'email' => 'SonataAdminBundle:CRUD:list_email.html.twig',
```

Comment:
> I say it would be a BC-break if the output was relied on by some program, but this is a listing, so only humans will be affected, and they will probably be thankful to have a link instead. 👍 to merge this in 3.x

same here 👍